opa/Dockerfile

```diff
@@ -107,7 +121,8 @@ COPY opa/licenses /licenses
 USER stackable
 WORKDIR /stackable/opa
 
-COPY --from=opa-builder /opa/opa /stackable/opa/opa
+COPY --from=opa-builder /opa/opa_${PRODUCT}.cdx.json /stackable/opa/
```

Comment:
Ah, this was wrong in the Airflow and Kafka Dockerfiles, I fixed it there. The idea was that we want to place the SBOM file where the application itself is located.

This is not important at all right now, but it's one extra way to determine what exactly this SBOM file is for (apart from the filename and the SBOM contents). It allows to draw a relation based on the filesystem hierarchy.

It might also come in handy if we create extra SBOMs for nested components, e.g. the SBOM for Trino would be located higher in the filesystem hierarchy (`/stackable/trino-server-451/trino-server-451.cdx.json`) than the SBOM for a specific Trino plugin (`/stackable/trino-server-451/plugin/whatever/whatever-123.cdx.json`).